Win10.ps1

```diff
@@ -29,6 +29,8 @@ $tweaks = @(
 	# "SetP2PUpdateLocal",          # "SetP2PUpdateInternet",       # "SetP2PUpdateDisable",
 	"DisableDiagTrack",             # "EnableDiagTrack",
 	"DisableWAPPush",               # "EnableWAPPush",
+	"HideRecentlyAddedApps",        # "ShowRecentlyAddedApps",

```

Comment:
I don't think that hiding recently added apps qualifies as privacy tweak. The apps will still be visible in the start menu, only they won't be pushed to the top. I'd put this to *UI tweaks*, probably after `Disable`/`EnableNewAppPrompt` so we have the "new app" stuff together.